content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -101,6 +101,15 @@ across namespaces, you need to use the fully qualified domain name (FQDN).
 As a result, all namespace names must be valid
 [RFC 1123 DNS labels](/docs/concepts/overview/working-with-objects/names/#dns-label-names).
 
+{{< warning >}}
+By creating namespaces with the same name as [public
+TLDs](https://data.iana.org/TLD/tlds-alpha-by-domain.txt), services in these
+namespaces can have short DNS names that overlap with public DNS names.
+Workloads from any namespace performing a DNS lookup without a trailing dot will
+be redirected to those services. For this and many other reasons, creating
+namespaces is a highly-privileged operation and should be done with care.
```

Comment:
How about:
```suggestion
be redirected to those services, which take precedence over DNS records in the
wider public internet.

To mitigate this, limit privileges for creating namespaces to trusted users. You could also
configure third-party security controls, such as
[admission webhooks](/docs/reference/access-authn-authz/extensible-admission-controllers/)
to block creating any namespace with the same name as a top-level domain
in the public internet.
```

Reply:
That would be a great idea, I didn't really know if it was a good place to write mitigation options!

Reply:
Feel free to accept that suggestion, or to make an equivalent change with your own preferred wording.

If you can, also [squash the commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits) in this PR.

Reply:
Thanks for the help 😊!